content/en/blog/2024/scaling-opentelemetry-collectors.md

```diff
@@ -0,0 +1,212 @@
+
+1. Create your hosts and add public SSH keys to them.
+
+  This example uses eight Linux hosts: two Ubuntu hosts, two CentOS hosts, two Fedora hosts, and two Debian hosts.
+
+2. Create an Ansible inventory file.
+
+  The Ansible inventory, which resides in a file named `inventory`, looks similar to this:
+
+  ```
+  146.190.208.216    # hostname = ubuntu-01
+  146.190.208.190    # hostname = ubuntu-02
+  137.184.155.128    # hostname = centos-01
+  146.190.216.129    # hostname = centos-02
+  198.199.82.174     # hostname = debian-01
+  198.199.77.93      # hostname = debian-02
+  143.198.182.156    # hostname = fedora-01
+  143.244.174.246    # hostname = fedora-02
+  ```
+
+  > **Note**: If you are copying the above file, remove the comments (#).
+
+3. Create an `ansible.cfg` file within the same directory as `inventory`, with the following values:
+  ```
+  [defaults]
+  inventory = inventory  # Path to the inventory file
+  private_key_file = ~/.ssh/id_rsa   # Path to my private SSH Key
+  remote_user=root
+  ```
+
+## Use the OpenTelemetry Collector Ansible role
+
+Next, you'll define an Ansible playbook to apply your chosen or created OpenTelemetry Collector role across your hosts.
+
+Create a file named `deploy-opentelemetry.yml` in the same directory as your `ansible.cfg` and `inventory`. 
+
+```yaml
+- name: Install OpenTelemetry Collector
+  hosts: all
+  become: true
+
+  vars:
+    grafana_cloud_api_key: <Your Grafana.com API Key>        # Example - eyxxxxxxxx
+    metrics_username: <prometheus-username>                  # Example - 825019
+    logs_username: <loki-username>                           # Example - 411478
+    prometheus_url: <prometheus-push-url>                    # Example - https://prometheus-us-central1.grafana.net/api/prom/push
+    loki_url: <loki-push-url>                                # Example - https://logs-prod-017.grafana.net/loki/api/v1/push
+    tempo_url: <tempo-push-url>                              # Example - tempo-prod-04-prod-us-east-0.grafana.net:443
+    traces_username: <tempo-username>                        # Example - 411478
+
+  tasks:
+    - name: Install OpenTelemetry Collector
+      ansible.builtin.include_role:
+        name: grafana.grafana.opentelemetry_collector
+      vars:
+        otel_collector_extensions:
+          basicauth/grafana_cloud_tempo:
+            # https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/extension/basicauthextension
+            client_auth:
+              username: "{{ traces_username }}"
+              password: "{{ grafana_cloud_api_key }}"
+          basicauth/grafana_cloud_prometheus:
+            client_auth:
+              username: "{{ prometheus_url }}"
+              password: "{{ grafana_cloud_api_key }}"
+          basicauth/grafana_cloud_loki:
+            client_auth:
+              username: "{{ logs_username }}"
+              password: "{{ grafana_cloud_api_key }}"
```

Comment:
```suggestion
```

Make this config independent of Grafana Cloud, we want to have this generally applicable.